src/components/cluster/detail/node_pool_dropdown_menu.js

```diff
@@ -2,71 +2,73 @@ import DropdownMenu from 'UI/dropdown_menu';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-const NodePoolDropdownMenu = props => (
-  <DropdownMenu
-    render={({
-      isOpen,
-      onClickHandler,
-      onFocusHandler,
-      onBlurHandler,
-      onKeyDownHandler,
-    }) => (
-      <div onBlur={onBlurHandler} onFocus={onFocusHandler}>
-        <button
-          aria-expanded={isOpen}
-          aria-haspopup='true'
-          onClick={onClickHandler}
-          onKeyDown={onKeyDownHandler}
-          type='button'
-        >
-          •••
-        </button>
-        {isOpen && (
-          <ul aria-labelledby='node_pools_dropdown' role='menu'>
-            <li>
-              <a
-                href='#'
-                onClick={e => {
-                  e.preventDefault();
-                  props.triggerEditName;
-                }}
-              >
-                Rename
-              </a>
-            </li>
-            <li>
-              <a
-                href='#'
-                onClick={e => {
-                  e.preventDefault();
-                  props.showNodePoolScalingModal(props.nodePool);
-                }}
-              >
-                Edit scaling limits
-              </a>
-            </li>
-            <li>
-              <a
-                href='#'
-                onClick={e => {
-                  e.preventDefault();
-                  props.nodePoolDelete(props.clusterId, props.nodePool);
-                }}
-              >
-                Delete
-              </a>
-            </li>
-          </ul>
-        )}
-      </div>
-    )}
-  />
-);
+const NodePoolDropdownMenu = props => {
+  return (
+    <DropdownMenu
+      render={({
+        isOpen,
+        onClickHandler,
+        onFocusHandler,
+        onBlurHandler,
+        onKeyDownHandler,
+      }) => (
+        <div onBlur={onBlurHandler} onFocus={onFocusHandler}>
+          <button
+            aria-expanded={isOpen}
+            aria-haspopup='true'
+            onClick={onClickHandler}
+            onKeyDown={onKeyDownHandler}
+            type='button'
+          >
+            •••
+          </button>
+          {isOpen && (
+            <ul aria-labelledby='node_pools_dropdown' role='menu'>
+              <li>
+                <a
+                  href='#'
+                  onClick={e => {
```

Comment:
What do you think about extracting these lambdas outside the `NodePoolDropdownMenu` component?

Reply:
Yeah, we can do it. But why?

Reply:
Just for better readibility and for easier editing

Reply:
When there is so little code I like to inline these functions. If the function has to grow for whatever reason, then I would write a named function.

Please, feel free to open a PR in `fmt` as a way to start a discussion for this if you think it's worth it.